### PR TITLE
Deavina_Assignment_21_Completed_25min

### PR DESCRIPTION
**Pull request by DeavinaSJun20**
image
image

### CODE REVIEW

File: src/deavinaSJun20/FibbonacciSeries.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	public static void main(String[] args) {
+		Scanner scanner = new Scanner(System.in);
+		System.out.println("Please enter input to display first n Fibonaaci Numbers: ");
+		int num = scanner.nextInt();
+		new FibbonacciSeries().fibbonacciSeries(num);
+		scanner.close();
+	}
+
+	private void fibbonacciSeries(int num) {
+		int firstnum = 0;
```
Comment (JagadeeshMJun20) [inline src/deavinaSJun20/FibbonacciSeries.java]: Fibbonacci series starts from 1. So please change your firstnum to 1

**Comment by DeavinaSJun20** [comment]
image

File: src/deavinaSJun20/FibbonacciSeries.java (hunk 1 above)
Comment (DeavinaSJun20) [reply src/deavinaSJun20/FibbonacciSeries.java]: we can start it from 0 also.
I have updated the code as per the review comments. Thanks

Comment (JagadeeshMJun20) [reply src/deavinaSJun20/FibbonacciSeries.java]: Now the code looks good to me.

**Review by JagadeeshMJun20 — APPROVED** [review]
Code is fine

**Pull request merged**